recipes/msix/all/patches/1.7/cmake.patch

```diff
@@ -0,0 +1,488 @@
+ CMakeLists.txt              | 102 +++---------------------------------
```

Comment:
This patch is doing too much, and will be a nightmare to maintain.
When patching files, try to keep the changes to a minimum.

Reply:
[Reduced the patch size](https://github.com/conan-io/conan-center-index/pull/6475/commits/a3144aa600399cfd7cd535b10d847ed80b658bed)